src/renderer/redux/actions/publish.js

```diff
@@ -119,16 +119,17 @@ export const doUploadThumbnail = (filePath: string, nsfw: boolean) => (
     body: data,
   })
     .then(response => response.json())
-    .then(json =>
-      json.success
-        ? dispatch({
-            type: ACTIONS.UPDATE_PUBLISH_FORM,
-            data: {
-              uploadThumbnailStatus: THUMBNAIL_STATUSES.COMPLETE,
-              thumbnail: `${json.data.url}${fileExt}`,
-            },
-          })
-        : uploadError(json.message)
+    .then(
+      json =>
+        json.success
+          ? dispatch({
+              type: ACTIONS.UPDATE_PUBLISH_FORM,
+              data: {
+                uploadThumbnailStatus: THUMBNAIL_STATUSES.COMPLETE,
+                thumbnail: `${json.data.url}${fileExt}`,
+              },
+            })
+          : uploadError(json.message)
```

Comment:
Not sure why we keep seeing these format changes when I didn't touch this stuff :(

Reply:
It's probably that atom plugin... I warned you of linter hell! 😆

Reply:
Hm good point. I'll explore